src/v2/view-builder/views/consent/ConsentAgreementText.js

```diff
@@ -0,0 +1,12 @@
+import { View } from 'okta';
```

Comment:
isn'n worth to be separated file?

Reply:
We should be striving for 1:1 view to file because it makes code easier to read.